articles/quickstart/spa/react/01-login.md

```diff
@@ -126,25 +126,32 @@ The Auth0 React SDK helps you retrieve the [profile information](https://auth0.c
 import React from "react";
 import { useAuth0 } from "@auth0/auth0-react";
 
-const Profile = () => {
+const UserProfile = () => {
```

Comment:
What's the reason for the component name change?

I'm ok with it but there are references to the old name in the surrounding text, which must also be updated.

Reply:
What's the reason for the component name change?

No reason at all. I have no specific preference for it.

Reply:
I just realized that was the name I gave for my own custom component, he he :P It's funny. 

Please keep it how it was before. Dont change the name. Let me fix that.

Reply:
Alright, I have put back the old component change. My bad again. I should not have put it in the first place (sorry, I just started learning react only a few weeks ago).

But, I am using Auth0 on my API server, so need to use it with my react learning process. 

Thank you.

Reply:
This still isn't changed. You only changed line 153 back to the original value.

Reply:
@stevehobbsdev @frederikprijck 

Folks, 

I am really new to react. any way you wish to resolve this, please do. Really, I don't mind.

Do, what is best but yeah, stop it from returning false is all I ask. If you can. 

Thank you.

Reply:
Thanks @Jay-study-nildana, you mentioned you are new to React and we understand the spirit of the change you're trying to make so we can take it from here and change the code.

Let's go with [the suggestion here](https://github.com/auth0/docs/pull/9548#discussion_r558601008). @frederikprijck can you please modify the PR to make this change when you have some time? If you're unable to change the PR, please raise a new one and link it here.